docs/modules/servers/pages/distributed/operate/guide.adoc

```diff
@@ -357,7 +357,7 @@ echo -e "\nauthorizer: org.apache.cassandra.auth.CassandraAuthorizer" >> /etc/ca
 ==== Create a role
 
 Have a look at
-http://cassandra.apache.org/doc/3.11.10/cql/security.html[cassandra documentation] section `CREATE ROLE` for more information
+http://cassandra.apache.org/doc/3.11.11/cql/security.html[cassandra documentation] section `CREATE ROLE` for more information
```

Comment:
I would say the version used is 3.11.10 and not 3.11.11, but indeed 3.11.10 is not found... weird? Anyway it's fine then :)